src/plugins/media-analysis/media-analysis.yml

```diff
@@ -18,5 +18,8 @@ mediaAnalysis:
     en: You can drop a CSV file in this box in order to import segments
     de: Sie können eine CSV-Datei in diese Box ziehen, um Segmente zu importieren
   startPlaybackFromHere:
-    en: Play from this part
-    de: Ab diesem Teil abspeilen
+    en: Play from this segment
+    de: Ab diesem Segment abspeilen
+  chapterPointerTooltip:
+    en: View segment text
+    de: Segment-Text sehen
```

Comment:
It sounds a little bit awkward, better would be "Segment-Text anzeigen" (or, as we call this text "annotation": "Anmerkungen zu diesem Segment anzeigen" / "Display annotations for this segment"?)

Reply:
How about "Display segment annotation"/ "Segment-Anmerkung anzeigen"  ? In singular, because it's one annotation field per chapter, and shorter phrased, more fitting a tooltip.

Reply:
That sounds good :+1: